README.md

```diff
@@ -85,34 +85,27 @@ In addition to `host` and `port`, you can also pass the following arguments:
 > Visit the any-agent [documentation](https://mozilla-ai.github.io/any-agent/) for more details on the supported
 > frameworks.
 
-#### Using Docker
+#### Using Makefile
```

Comment:
I would keep it as `Using Docker`, because we may want to use the Makefile to run the agent as a process as well.